src/main/java/emu/grasscutter/net/packet/PacketOpcodesUtils.java

```diff
@@ -27,7 +27,10 @@ public class PacketOpcodesUtils {
         PacketOpcodes.PingRsp,
         PacketOpcodes.WorldPlayerRTTNotify,
         PacketOpcodes.UnionCmdNotify,
-        PacketOpcodes.QueryPathReq
+        PacketOpcodes.QueryPathReq,
+        PacketOpcodes.QueryPathRsp,
+        PacketOpcodes.PlayerPropNotify,
```

Comment:
Probably better to leave `PlayerPropNotify` and `PlayerPropChangeNotify` out, they are useful for debugging in ways the rest of the list aren't.

Reply:
Deleted, keep only PacketOpcodes.QueryPathRsp